public/config/configuration.json

```diff
@@ -18,5 +18,8 @@
   "SENTRY_ENABLE": "false",
   "SENTRY_DSN": "",
   "STATUS_API_URL": "https://status-api-dev.apps.silver.devops.gov.bc.ca",
-  "STATUS_API_VERSION": "/api/v1"
+  "STATUS_API_VERSION": "/api/v1",
+  "BUSINESS_SEARCH_API_URL": "https://bcregistry-dev.apigee.net/registry-search",
+  "BUSINESS_SEARCH_API_VERSION": "/api/v1",
+  "BUSINESS_SEARCH_API_KEY": "KHGuiTxG52ejzFZi7wu4QyD8GdVGEMAz"
```

Comment:
Do these exist in 1Password, and is the vaults file configured, so that these new keys will exist in Dev/Test/Prod?

Reply:
If anything needs to be done during deployment, please add notes to the release ticket(s).

Reply:
I don't think it is. Will check with Thor